cpp/tests/c_api/mg_triangle_count_test.c

```diff
@@ -46,21 +46,28 @@ int generic_triangle_count_test(const cugraph_resource_handle_t* handle,
   cugraph_type_erased_device_array_t* p_start           = NULL;
   cugraph_type_erased_device_array_view_t* p_start_view = NULL;
 
+  int rank = 0;
+  rank = cugraph_resource_handle_get_rank(handle);
```

Comment:
Sorry for nitpicking, but `int rank = cugraph_resource_handle_get_rank(handle);` might be cleaner.

Reply:
Fixed